casa/OS/HostInfo.h

```diff
@@ -131,12 +131,22 @@ class HostInfo
     static ptrdiff_t swapFree();
     // </group>
 
+    // Introduce run-time memory limit re-specification capabilities
+    // <group>
+    static void setMemoryTotal(int memory);
```

Comment:
its probably useful to return the old value in the setter to simplify reverting the setting again